pkg/catalogv2/helm/release.go

```diff
@@ -58,6 +60,7 @@ func IsLatest(release *v1.ReleaseSpec, others []runtime.Object) bool {
 	return true
 }
 
+// ToRelease return a v1.ReleaseSpec for the given runtime.Object received
```

Comment:
// ToRelease return a v1.ReleaseSpec for the given runtime.Object received.
// The function populates the releaseSpec with release information related
// to an installed app based on the helm version used to install the app.